functions/gmail-notifier/index.js

```diff
@@ -76,6 +77,18 @@ exports.initWatch = async (req, res) => {
   return res.send(`Watch initialized on gmail inbox: ${email}`)
 }
 
+exports.postMessageToSlack = async (text) => {
+  const token = 'slack-bot-token'
+  const web = new WebClient(token)
+  const conversationID = 'C021BGBSMFY'
```

Comment:
this should be in some configuration file

Reply:
Agreed, should be moved into GCP secret i think